cypress/support/fragments/requests/requests.js

```diff
@@ -314,10 +315,20 @@ export default {
   },
 
   filterRequestsByTag(tag) {
+    cy.wait(2000);
     cy.do(
       Pane({ title: 'Search & filter' })
         .find(MultiSelect({ ariaLabelledby: 'tags' }))
-        .select(tag),
+        .choose(tag),
+    );
+  },
+
+  clearSelectedTag() {
```

Comment:
method will failed if select more than 1 tag
btw in test you need click 'xButton' next to Tags accordion
see resetTypeOfHeading() method as example